pubspec.yaml

```diff
@@ -7,10 +7,7 @@ dependencies:
   flutter:
     sdk: flutter
   collection: ^1.16.0
-  sprintf: ^6.0.2
   timezone: ^0.9.0
-  flutter_native_timezone: ^2.0.0
-  intl: ^0.17.0
```

Comment:
are these three dependencies not used anymore or why can they be removed?

Reply:
These are used in the example app, but not really used in the package itself.

Reply:
Shouldn't we add them as dependencies to the `example/pubspec.yaml` then?

Maybe I misunderstood, but as far as I can tell, now the example app cannot access those three packages anymore (haven't tested the example app tho).

Reply:
They're already there lol
No one bothered to clean it up before I think

Reply:
You're right, `intl` and `flutter_native_timezone` are already there. `sprintf` is missing, but never imported, thus not necessary.